src/System.ServiceProcess.ServiceController/tests/TestServiceProvider.cs

```diff
@@ -90,22 +127,12 @@ public void DeleteTestServices()
         {
             try
             {
+                if (_client != null)
+                    _client.Dispose();
```

Comment:
@Anipik it's not worth changing now, but generally after Dispose you would set _client = null.